recaf-ui/src/main/java/me/coley/recaf/ui/pane/MethodCallGraphPane.java

```diff
@@ -172,7 +172,7 @@ protected void updateItem(MethodInfo item, boolean empty) {
 					removeEventFilter(MouseEvent.MOUSE_PRESSED, onClickFilter);
 			} else {
 				onClickFilter = null;
-				Text classText = new Text(TextDisplayUtil.escapeShortenPath(item.getOwner()));
+				Text classText = new Text(EscapeUtil.escape(item.getOwner()));
```

Comment:
Was renamed to `shortenEscapeLimit`.

Should use `TextDisplayUtil.shortenEscapeLimit(...)` for consistency of using `TextDisplayUtil` when possible _(and limiting for stupid long names from obfuscators)_